test/gst-msdk/util.py

```diff
@@ -23,7 +23,7 @@ def get_supported_format_map():
     "I420"  : ("i420", "I420"),
     "NV12"  : ("nv12", "NV12"),
     "YV12"  : ("yv12", "YV12"),
-    "AYUV"  : ("ayuv", "AYUV"),
+    "AYUV"  : ("vuya", "VUYA"),
     "YUY2"  : ("yuy2", "YUY2"),
```

Comment:
I think you should add "VUYA" : ("vuya", "VUYA") here, then replace AYUV with VUYA in the caps files.

Reply:
agree.

Reply:
ayuv mapping should not be removed here.  It may still be useful in other cases.

Reply:
Thanks for your advice,I will modify it soon.

Reply:
One more thing... you should also add VUYA to the lib/formats.py subsampling map so tests can add software conversion if needed.

Reply:
And to be clear, the caps files specify the supported hardware formats whereas the test/gst-msdk/util.py specify the supported middleware formats.

Reply:
I want to add "VUYA" to where exist "AYUV".Can I use this means to deal with this question?

Reply:
Consider this example:

A user config can define a test case where the source and/or reference is defined in AYUV format.  When executed, the test function will query the hardware/driver caps file and find only VUYA is supported.  Then, the test will query for a compatible subsampling format (in lib/formats.py) and verify both are supported by middleware.  For raw input, the test will insert a software conversion from AYUV -> VUYA for hardware upload.  For raw outputs, it will insert a software conversion from hardware downloaded VUYA -> AYUV so that it can do the comparison.

Reply:
I have modify my patch to add "VUYA".But I find that I test gst-msdk/gst-vaapi decode by vaapi-fits,the function "read_frame_VUYA" which I add in file "lib/framereader.py " will not execute.I wonder if it is neccessary for us to add it?

Reply:
the frame readers are only needed for output metrics.